nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/ImagingComponent.java

```diff
@@ -25,7 +25,7 @@
 import static org.nuxeo.ecm.platform.picture.api.ImagingConvertConstants.OPTION_RESIZE_HEIGHT;
 import static org.nuxeo.ecm.platform.picture.api.ImagingConvertConstants.OPTION_RESIZE_WIDTH;
 
-import java.awt.Point;
+import java.awt.*;
```

Comment:
we dont use import all classes of a given package (`*`) you should configure you IDE, https://doc.nuxeo.com/corg/intellij-idea-setup/